### PR TITLE
Update rank graphics

### PR DESCRIPTION
**Pull request by frenzibyte**
- [x] ppy/osu-resources#58

### CODE REVIEW

File: osu.Game/Scoring/ScoreRank.cs [hunk 1]
```diff
@@ -7,10 +7,10 @@ namespace osu.Game.Scoring
 {
     public enum ScoreRank
     {
-        [Description(@"F")]
+        [Description(@"D")]
```
Comment (Aergwyn) [inline osu.Game/Scoring/ScoreRank.cs]: If `F` apparently doesn't exist, can this be deleted completely?

Comment (iiSaLMaN) [reply osu.Game/Scoring/ScoreRank.cs]: yes, that's what i did in ppy/osu-resources#58

Comment (Aergwyn) [reply osu.Game/Scoring/ScoreRank.cs]: Then why is it still here?

Comment (iiSaLMaN) [reply osu.Game/Scoring/ScoreRank.cs]: oh, you meant the `ScoreRank.F`

**Review by peppy — APPROVED** [review]

**Pull request merged**